contractcourt/channel_arbitrator.go

```diff
@@ -573,9 +574,11 @@ func (c *ChannelArbitrator) Start(state *chanArbStartState) error {
 		}
 	}
 
-	c.wg.Add(1)
-	go c.channelAttendant(bestHeight)
-	return nil
+	err = c.wg.Go(func(ctx context.Context) {
```

Comment:
nit: could return directly here.

Reply:
So what does this do exactly? No error is returned from `channelAttendant`. 

Was this issue actually introduced by adding the a goroutine the prior commit?